opensearch-operator/Dockerfile.oracle

```diff
@@ -0,0 +1,31 @@
+# Build the manager binary
+FROM --platform=$BUILDPLATFORM golang:1.19 as builder
```

Comment:
This should also be oraclelinux, you'd need to get golang from a microdnf repo

Reply:
Yes. You're getting packages (e.g. golang) from a thirdparty base image which needs to be changed to become oracle compliant.

Reply:
updated